python-avd/schema_tools/build_schemas.py

```diff
@@ -48,9 +52,15 @@ def combine_schemas() -> None:
 
 def validate_schemas(schema_store: dict) -> None:
     """Validate schemas according to metaschema."""
+    if not HAS_JSONSCHEMA_RS:
+        LOGGER.warning(
+            "'jsonschema_rs' was not found. If you are a developer using AVD make sure to install the dev requirements for the collection. "
+            "The schemas could not be validated."
+        )
```

Comment:
@ClausHolbechArista let me know if the wording is ok here

Reply:
ok